test/lib/ufe/testCamera.py

```diff
@@ -231,16 +231,73 @@ def _TestProjection(self, ufeCamera, usdCamera):
 
     @unittest.skipUnless(mayaUtils.mayaMajorVersion() >= 2023, 'Requires Python API only available in Maya 2023 or greater.')
     def testUsdCamera(self):
-        self._StartTest('TranslateRotate_vs_xform')
         mayaPathSegment = mayaUtils.createUfePathSegment('|stage|stageShape')
         cam2UsdPathSegment = usdUtils.createUfePathSegment('/cameras/cam2/camera2')
-        cam2Path = ufe.Path([mayaPathSegment, cam2UsdPathSegment])
-        cam2Item = ufe.Hierarchy.createItem(cam2Path)
+        camera2Path = ufe.Path([mayaPathSegment, cam2UsdPathSegment])
+        cam2Item = ufe.Hierarchy.createItem(camera2Path)
 
         cam2Camera = ufe.Camera.camera(cam2Item)
         cameraPrim = usdUtils.getPrimFromSceneItem(cam2Item)
         self._TestCameraAttributes(cam2Camera, cameraPrim)
 
+    @unittest.skipUnless(os.getenv('UFE_PREVIEW_VERSION_NUM', '0000') >= '4011', 'Requires Ufe Preview Version at least 2011.')
+    def testUsdCameraHandler(self):
+        mayaPathSegment = mayaUtils.createUfePathSegment('|stage|stageShape')
+        cam2UsdPathSegment = usdUtils.createUfePathSegment('/cameras/cam2/camera2')
+        camera2Path = ufe.Path([mayaPathSegment, cam2UsdPathSegment])
+
+        # Test that the camera handlers can find USD cameras in a scene segment
+        proxyShapeParentSegment = mayaUtils.createUfePathSegment('|stage')
```

Comment:
I wonder about these single-use UFE path segments --- can't we just use ufe.PathString.path() to create them out of concatenated strings?  E.g.
mayaPathSegmentStr = '|stage|stageShape'
camerasParentPath = ufe.PathString.path(mayaPathSegmentStr+'/cameras/cam2')
I get that you're re-using the Maya path segment, and in real code there may be a performance benefit to doing this, but in a test it seems so verbose.